src/Collapse/Collapse.component.tsx

```diff
@@ -53,9 +55,10 @@ export const Collapse = ({
     isOpen = false,
     openText = DEFAULT_OPEN_TEXT,
     closeText = DEFAULT_CLOSE_TEXT,
-    openIcon = false,
-    closeIcon = false,
+    openIcon,
+    closeIcon,
     theme = DEFAULT_THEME,
+    toggleFn,
```

Comment:
I would rename to something more conventional like `onClick` or `onToggle`.

Reply:
I don't like reusing event names like onClick, and I'm not sure onToggle quite fits either since that sounds like an event. toggleFn is a function that controls the toggling of open/close of a Collapse child. It could potentially use something more descriptive, but I'd like it to be more representative of that. Ideas?

Reply:
Well... it is an event though. It *isn't* a function that controls the toggling of the collapse, it is really just a function that runs when the button is clicked. It's up to the parent to decide what that function does.

I suggested just `onToggle` initially, but then I remembered https://jxnblk.com/blog/defining-component-apis-in-react/#keep-props-apis-parallel and added `onClick` as my primary suggestion

Reply:
I don't agree that it's an event. An event is `onClick`, `onMouseOver` etc., right? What happens when those events happen is an event handler. Maybe that's the answer though, what about `onToggleHandler` or something similar?

Reply:
Well yes, it is an event *handler* for the onClick event. This is typically how react handler props are defined however.
```jsx
<button onClick={handleButtonClick} ></button>
<Collapse onClick={() => toggleThisGuyAndCloseEveryOneElse()} />
```

The reason I suggest this change is actually because I was having difficulty understanding what the CollapseGroup was doing until I realized that the `toggleFn` prop is just the onClick handler. And in the case of the Accordion, the behavior the Collapse when clicked is to close all other collapses.

Reply:
Yep, I think we're in agreement now. I'll change the wording both in `Collapse` and `CollapseGroup` to try and make it more clear that `CollapseGroup` is passing a new handler to `Collapse` to use. Good discussion, thanks!

Reply:
Sorry to be late to this party, this is a solid discussion.

### RE Event Handling
I think the react team has some opinions on event handling [naming conventions](https://reactjs.org/docs/events.html). It seems that they rely on a `on` prefix denoting that this is an event handler, an event name `on[EventName]`, and then a modifier for meta (think pre/post) event handlers `on[EventName][Modifier]` so for a bubbling-handler `onClickCapture` works. Then, inside the component itself, they use `handle[EventName]` for the internal component method handling the event.

### RE Event-Handling By Reference
I can't find any opinionated naming conventions around this, but the method being described here is a trigger, so what if we all named instance trigger methods with a `trigger` prefix? `triggerCollapse(true)` to open or `triggerCollapse(false)` to close? I'm not dead set on `trigger` so much as I'd like to define a consistent naming convention for these types of events to that in our docs anytime someone sees a keyword they will intuit that it's a trigger-method for controlling a component instance.

Reply:
Looking more closely at the code and the [readme](https://github.com/RetailMeNot/anchor/pull/54/files#diff-6fdb9d96efcd884c4398b16c8de687ab) (which may not have been updated with this feedback), it doesn't look like this is a trigger. It doesn't allow an external component to trigger events, it's an internal event handler, so I think this is more of a traditional event handler (i.e. `on[EventName]`) than a trigger.